packages/appstore/src/components/fiat-transaction-list/fiat-transaction-list.tsx

```diff
@@ -25,6 +25,7 @@ const FiatTransactionList = () => {
         account_type: 'binary',
         balance: 0,
         currency: 'BTC',
+        gradient_for: 'btc',
```

Comment:
@hamid-deriv Why did you add new key(gradient_for) since we have the same key with uppercase characters?

Reply:
@mahdiyeh-deriv Because for a demo account, the currency is USD but we need to have a demo instead of USD. 
I add this just to handle the conflicts between USD and the demo account.